frontend/javascripts/oxalis/controller.js

```diff
@@ -114,6 +118,13 @@ class Controller extends React.PureComponent<PropsWithRouter, State> {
       .catch(error => {
         this.props.setControllerStatus("failedLoading");
         const isNotFoundError = error.status === 404;
+        if (isNotFoundError) {
+          // The dataset could probably not be fetched, because the user is currently not in the correct organization.
+          // Thus fetch them to check whether the user can switch to the organization of the dataset.
```

Comment:
just nitpicking

```suggestion
          // The dataset could not be fetched. This might have happened because the user is currently not in the correct organization.
          // Thus, fetch the user's organizations to check whether they can switch to the organization of the dataset.
```